src/NuGet.Server.Core/Core/PackageFactory.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information. 
+
+using System;
+using System.IO;
+
+namespace NuGet.Server.Core
+{
+    public static class PackageFactory
+    {
+        public static IPackage Open(string fullPackagePath)
+        {
+            if (fullPackagePath == null)
```

Comment:
string.IsNullOrEmpty instead of just null?

Reply:
Sure. Fixed.